### PR TITLE
Fully implement cyclictest_wrapper

### PR DESCRIPTION
**Pull request by smalleni**
Signed-off-by: Sai Sindhur Malleni <smalleni@redhat.com>

### CODE REVIEW

**Comment by smalleni** [comment]
@jtaleric @rsevilla87 @dry923 PTAL

Link to doc: https://search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com/_plugin/kibana/app/kibana#/discover?_g=(refreshInterval:(pause:!t,value:0),time:(from:'2020-10-23T17:30:00.000Z',to:now))&_a=(columns:!(_source),index:'6a23f240-155b-11eb-b44a-2b4ae93882bf',interval:auto,query:(language:kuery,query:''),sort:!(timestamp,desc))

File: snafu/cyclictest_wrapper/trigger_cyclictest.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+class Trigger_cyclictest():
+    def __init__(self, args):
+        self.uuid = args.uuid
+        self.user = args.user       
+        self.path = args.path
+        self.samples = args.samples
+
+        self.stressng = args.stressng
+        self.duration = args.duration
+        self.disable_cpu_balance = args.disable_cpu_balance
```
Comment (rsevilla87) [inline snafu/cyclictest_wrapper/trigger_cyclictest.py:32]: These variables are not used anywhere

Comment (smalleni) [reply snafu/cyclictest_wrapper/trigger_cyclictest.py:32]: Ah, I should probably add these to the JSON payload.

Comment (smalleni) [reply snafu/cyclictest_wrapper/trigger_cyclictest.py:32]: Done

File: snafu/cyclictest_wrapper/cyclictest_wrapper.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        parser.add_argument(
+            '-s', '--samples', type=int, default=1,
+            help='Number of times to run the benchmark')
+        parser.add_argument(
+            '-u', '--uuid',
+            required=True,
+            help='Provide the uuid')
+        parser.add_argument(
+            '--user',
+            required=True,
```
Comment (dry923) [inline snafu/cyclictest_wrapper/cyclictest_wrapper.py:40]: Isn't required when providing a default a bit contradictory? I.e. if theres a default so I shouldn't have to provide a value

Comment (smalleni) [reply snafu/cyclictest_wrapper/cyclictest_wrapper.py]: Fixed

File: snafu/utils/wrapper_factory.py [hunk 3]
```diff
@@ -11,7 +11,11 @@
 from snafu.ycsb_wrapper.ycsb_wrapper import ycsb_wrapper
 from snafu.scale_openshift_wrapper.scale_openshift_wrapper import scale_openshift_wrapper
 from snafu.stressng_wrapper.stressng_wrapper import stressng_wrapper
+<<<<<<< HEAD
```
Comment (dry923) [inline snafu/utils/wrapper_factory.py:14]: @smalleni FYI extra bits from the rebase made it in

Comment (smalleni) [reply snafu/utils/wrapper_factory.py:14]: Serves me right for rebasing during a meeting.

File: snafu/utils/wrapper_factory.py [hunk 4]
```diff
@@ -29,6 +30,7 @@
     "scale": scale_openshift_wrapper,
     "stressng": stressng_wrapper,
     "upgrade": upgrade_openshift_wrapper
+    "cyclictest": cyclictest_wrapper
```
Comment (dry923) [inline snafu/utils/wrapper_factory.py:33]: need a comma the line above cyclictest

Comment (dry923) [reply snafu/utils/wrapper_factory.py:33]: ie at the end of the upgrade line, lol

**Review by dry923 — APPROVED** [review]
LGTM!

**Review by mohit-sheth — APPROVED** [review]
LGTM

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**